loader/sky-processor/index.js

```diff
@@ -0,0 +1,50 @@
+/*jshint node: true*/
+'use strict';
+
+const logger = require('winston');
+const skyPagesConfigUtil = require('../../config/sky-pages/sky-pages.config');
+const config = skyPagesConfigUtil.getSkyPagesConfig();
```

Comment:
The `skyPagesConfig` is already exposed to loaders as `this.options.SKY_PAGES`.  I would use this mechanism instead of calling `getSkyPagesConfig` again, otherwise processes further up the stack won't be able to make changes that are propagated.